src/Microsoft.ML.StandardLearners/Standard/LogisticRegression/MulticlassLogisticRegression.cs

```diff
@@ -104,7 +104,7 @@ public sealed class Options : OptionsBase
         internal MulticlassLogisticRegression(IHostEnvironment env, Options options)
             : base(env, options, TrainerUtils.MakeU4ScalarColumn(options.LabelColumn))
         {
-            ShowTrainingStats = Args.ShowTrainingStats;
+            ShowTrainingStats = LbfgsTrainerOptions.ShowTrainingStats;
```

Comment:
>LbfgsTrainerOptions [](start = 32, length = 19)

Same here why not naming this `Options`?

Reply:
Mostly for clarity.  Throughout the public API surface we have the class `Options`  to represent a particular concept.  

I wanted to name this property something different , so users can draw the distinction with the `Options` class

---
In reply to: [258279561](https://github.com/dotnet/machinelearning/pull/2636#discussion_r258279561) [](ancestors = 258279561)